sid/tests/test_pathogenesis.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pandas as pd
+from pandas.testing import assert_series_equal
+
+import sid.pathogenesis as pg
```

Comment:
Import ``_draw_countdowns`` explicitly. This makes it more obvious which functions are going to be tested.

Reply:
Done.